.agents/skills/prepare-release/SKILL.md

```
@@ -0,0 +1,37 @@
+---
+name: prepare-release
+description: 'Bump display version to user-provided version and push tag. No changelog, no GitHub Release.'
+user-invocable: true
+argument-hint: '[version] e.g. 1.2.0'
+---
+
+# Prepare Release
+
+Bumps `manifest.json` + `package.json` to version provided by user (`src/core/version.js` reads manifest for `version-display`) and pushes `vX.Y.Z` tag.
+
+## Procedure
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)